deploy/kubernetes/base/node_windows/node.yaml

```diff
@@ -0,0 +1,46 @@
+#TODO: Force DaemonSet to not run on master.
```

Comment:
Have we tested the driver to actually run on windows? I didn't think that filestore supported SMB mounts which is how windows does NFS (although I could totally be wrong).

Anyway if it's not tested, I'd prefer to leave out the windows stuff for a future PR.